package.json

```diff
@@ -23,7 +23,7 @@
         "@headlessui/tailwindcss": "^0.2.1",
         "@safe-global/safe-apps-sdk": "^9.1.0",
         "@sentry/nextjs": "^8.30.0",
-        "@squirrel-labs/peanut-sdk": "^0.5.8",
+        "@squirrel-labs/peanut-sdk": "^0.5.9",
```

Comment:
this will have to be 0.5.10 (already deployed one sdk release hotfix since this PR was  made). Also, this package update is not sufficeint, also have to update pnpm lock, so an extra commit is needed